msfvenom

```diff
@@ -43,7 +43,7 @@ def init_framework(create_opts={})
     type = Msf.const_get("MODULE_#{type.upcase}")
   end
 
-  @framework = ::Msf::Simple::Framework.create(create_opts.merge('DisableDatabase' => true))
+  @framework = ::Msf::Simple::Framework.create(create_opts.merge('DisableDatabase' => false))
```

Comment:
This would be good to split out as an independent pull request. I think you might be able to avoid having the create_opts part altogether and just say Framework.create by itself.